src/reference-manual/compiler-optimizations.Rmd

```diff
@@ -0,0 +1,487 @@
+The Stanc3 compiler can attempt to optimize a Stan program as it is compiled. The optimized program has the same behavior as the unoptimized program, but it may be faster, more memory efficient, or more numerically stable.
```

Comment:
```suggestion
The Stanc3 compiler can optimize the code of Stan model during compilation. The optimized model code behaves the same as unoptimized code, but it may be faster, more memory efficient, or more numerically stable.
```